app/src/main/java/org/oppia/android/app/activity/route/ActivityRouterModule.kt

```diff
@@ -0,0 +1,19 @@
+package org.oppia.android.app.activity.route
+
+import dagger.Module
+import dagger.Provides
+import dagger.multibindings.IntoMap
+import dagger.multibindings.StringKey
+import org.oppia.android.app.model.DestinationScreen
+
+/** RouterModule to bind [DestinationScreen]. */
```

Comment:
Ditto here--perhaps just say 'Module'?

Also, what's specifically being bound & how? The KDoc reads a bit as an incomplete thought.

Reply:
Updated RouterModule to Module in kdoc